### PR TITLE
Elgato.GameCaptureHD60SDriver version 1.1.0.194.3

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26822)

### CODE REVIEW

File: manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCaptureHD60SDriver
+PackageVersion: 1.1.0.194.3
+PackageLocale: en-US
+Publisher: Elgato Systems
+PackageName: Elgato Game Capture HD60 S Driver Installer
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.locale.en-US.yaml:8]: image
```suggestion
PackageName: Game Capture HD60 S
```

File: manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCaptureHD60SDriver
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.installer.yaml:4]:
```suggestion
PackageIdentifier: Elgato.GameCaptureHD60S
```

File: manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCaptureHD60SDriver
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Elgato.GameCaptureHD60S
```

File: manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCaptureHD60SDriver
```
Comment (OfficialEsco) [inline manifests/e/Elgato/GameCaptureHD60SDriver/1.1.0.194.3/Elgato.GameCaptureHD60SDriver.yaml:4]:
```suggestion
PackageIdentifier: Elgato.GameCaptureHD60S
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Remember to change the software folder name

File: manifests/e/Elgato/GameCaptureHD60S/1.1.0.194.3/Elgato.GameCaptureHD60S.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCaptureHD60S
+PackageVersion: 1.1.0.194.3
```
Comment (ItzLevvie) [inline manifests/e/Elgato/GameCaptureHD60S/1.1.0.194.3/Elgato.GameCaptureHD60S.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.1.0.194
```

File: manifests/e/Elgato/GameCaptureHD60S/1.1.0.194.3/Elgato.GameCaptureHD60S.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCaptureHD60S
+PackageVersion: 1.1.0.194.3
```
Comment (ItzLevvie) [inline manifests/e/Elgato/GameCaptureHD60S/1.1.0.194.3/Elgato.GameCaptureHD60S.yaml:5]:
```suggestion
PackageVersion: 1.1.0.194
```

File: manifests/e/Elgato/GameCaptureHD60S/1.1.0.194.3/Elgato.GameCaptureHD60S.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Elgato.GameCaptureHD60S
+PackageVersion: 1.1.0.194.3
```
Comment (ItzLevvie) [inline manifests/e/Elgato/GameCaptureHD60S/1.1.0.194.3/Elgato.GameCaptureHD60S.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.0.194
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**